src/lib/simulation/rewardsGuardian.ts

```diff
@@ -0,0 +1,54 @@
+import LootTable from 'oldschooljs/dist/structures/LootTable';
+
+const TalismanTable = new LootTable()
+	.add('Mind talisman', 1, 4)
+	.add('Body talisman', 1, 4)
+	.add('Chaos talisman', 1, 4)
+	.add('Cosmic talisman', 1, 4)
+	.add('Nature talisman', 1, 4)
+	.add('Air talisman', 1, 3)
+	.add('Water talisman', 1, 3)
+	.add('Earth talisman', 1, 3)
+	.add('Fire talisman', 1, 3)
+	.add('Elemental talisman', 1, 1)
+	.add('Law talisman', 1, 1)
+	.add('Death talisman', 1, 1);
+
+const AbyssalDyeTable = new LootTable().add('Abyssal red dye').add('Abyssal green dye').add('Abyssal blue dye');
+
+const RareRewardTable = new LootTable()
+	.oneIn(200, 'Catalytic talisman')
+	.oneIn(300, 'Abyssal needle')
+	.oneIn(700, 'Abyssal lantern')
+	.oneIn(400, AbyssalDyeTable);
+
+export const rewardsGuardianTable = new LootTable()
+	.every(RareRewardTable)
+
+	/* Runes */
+	.add('Chaos rune', [61, 150], 16)
+	.add('Cosmic rune', [20, 30], 16)
+	.add('Nature rune', [28, 150], 16)
+	.add('Law rune', [9, 120], 16)
+	.add('Death rune', [5, 120], 16)
+	.add('Blood rune', [5, 120], 16)
+	.add('Air rune', [400, 500], 16)
+	.add('Water rune', [400, 500], 16)
+	.add('Earth rune', [400, 500], 16)
+	.add('Fire rune', [400, 500], 16)
+	.add('Mind rune', [250, 400], 16)
+	.add('Body rune', [80, 150], 16)
+
+	/* Talismans */
+	.add(TalismanTable, 1, 16)
+
+	/* Other */
+	.add('Abyssal pearls', [10, 20], 18)
+	.add('Intricate pouch', 1, 5)
+	.add('Abyssal ashes', 1, 1)
+	.add('Needle', 1, 1)
+
+	/* Tertiary */
+	// Atlax moved here to not mess with rate but still drop for the sake of it.
```

Comment:
Atlax is normally only dropped once in game, if I had it in same loot table as wiki it would screw with the droprates so I just moved it to a tertiary to keep drop rates same.